interface/weno/confirm.php

```diff
@@ -0,0 +1,258 @@
+
+
+<?php
+/** Copyright (C) 2016 Sherwin Gaddis <sherwingaddis@gmail.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+ 
+$fake_register_globals=false;
+$sanitize_all_escapes=true;	
+include_once('../globals.php');
+include_once('transmitDataClass.php');
+include_once('$srcdir/patient.inc');
+
+$date = date("Y-m-d");
+$pid = $GLOBALS['pid'];
+$uid = $_SESSION['authUserID'];          //username of the person for this session
+
+$tData = new transmitData();
+
+$send = $tData->getDrugList($pid, $date);
+$provider = $tData->getProviderFacility($uid);
+$patientPharmacy = $tData->patientPharmacyInfo($pid);
+$mailOrder = $tData->mailOrderPharmacy();
+
+
+?>
+
+<html>
+<head>
+<?php html_header_show();?>
+<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-ui-1-10-4/themes/dot-luv/jquery-ui.min.css" type="text/css" />
```

Comment:
Remove lines 45, 46, and 123 . Then add following here(this will bring in the sunny jquery-ui theme):
<?php
$include_standard_style_js = array("jquery-ui-sunny");
require "{$GLOBALS['srcdir']}/templates/standard_header_template.php";
?>